### PR TITLE
EZP-29729: As an editor, I want to have access to all my drafts

### PR DESCRIPTION
**Pull request by adamwojs**
| Question      | Answer
| ------------- | ---
| Tickets       | https://jira.ez.no/browse/EZP-29729
| Bug fix?      | no
| New feature?  | yes
| BC breaks?    | no
| Tests pass?   | yes
| Doc needed?   | yes
| License       | [GPL-2.0](https://github.com/ezsystems/ezplatform-admin-ui/blob/master/LICENSE)

The only place when as an editor I can go back go to the editing of new content item draft is a "Dashboard" > "Me" > "Drafts".  Unfortunately, this is not possible when I have more than 10 drafts at once. 

This PR solves this issue by adding "My drafts" list in the user context menu (similar to the legacy administration panel).

screenshot_2018-10-20 drafts - ez platform
screenshot_2018-10-20 drafts - ez platform 1
screenshot_2018-10-20 ez platform

#### Checklist:
- [X] Coding standards (`$ composer fix-cs`)
- [X] Create JIRA issue
- [X] Align with InVision Prototype
- [x] Ready for Code Review

### CODE REVIEW

**Comment by andrerom** [comment]
Nice one @adamwojs, will there be link from dashboard to this as well when there is more then 10 drafts?

**Comment by lserwatka** [comment]
Yes, this is the idea.

File: src/bundle/Resources/views/admin/mydrafts/list.html.twig [hunk 1]
```diff
@@ -0,0 +1,125 @@
...
+{%- block page_title -%}
+    {% include '@ezdesign/parts/page_title.html.twig' with {
+        title: 'mydrafts.list'|trans|desc('My drafts'),
+        iconName: 'create-content'
+    } %}
+{%- endblock -%}
+
+{% block content %}
+    <div class="row align-items-stretch ez-main-row">
+        <section class="container mt-4">
```
Comment (inakijv) [inline src/bundle/Resources/views/admin/mydrafts/list.html.twig]: Please add offset `px-5` for consistency with the rest of views with tables

**Comment by inakijv** [comment]
Hi @adamwojs this would be the icon for drafts view.
[drafts.svg.zip](https://github.com/ezsystems/ezplatform-admin-ui/files/2480630/drafts.svg.zip)

**Comment by adamwojs** [comment]
PR updated according to  @mikadamczyk suggestions.

**Comment by lserwatka** [comment]
@adamwojs can we proceed with QA here?

**Comment by barbaragr** [comment]
1. Please change the message after clicking trash button for: `Are you sure you want to permanently delete the selected draft(s)?`

2. Version conflict window isn't displayed
a. Create CI and make about 3 drafts after that (v2, v3, v4)
b. Go to User menu/Drafts
c. Click third version (v3) of the CI and publish it
d. Go back to the Drafts list and click on edit button next to second version (v2)
Actual result: 
Console shows: `GET http://eev2/admin/version/has-no-conflict/67/3/eng-GB 409 (Conflict)
Uncaught (in promise) TypeError: Cannot set property 'href' of null`
Expected result: Version conflict window should appear I suppose.

**Comment by adamwojs** [comment]
PR updated according to @barbaragr suggestions 😉

**Pull request merged**